server/isomorphic-routing/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,61 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`getNormalizedPath should return expected results for a variety of inputs 1`] = `""`;
```

Comment:
Empty path `""` is being handled as `""`. I'm not sure whether we'll ever see that as an empty path is implicitly `/` as far as I know. We can probably drop this test.